client/src/resources/cert-and-project-map.ts

```diff
@@ -1,4 +1,5 @@
 import { SuperBlocks } from '../../../config/certification-settings';
+import { showUpcomingChanges } from '../../../config/env.json';
```

Comment:
It's probably fine like this, but for consistency with tsx files (where it does seem to matter) can we `import envData` and then destructure `showUpcomingChanges` from that?